modules/genomic_browser/templates/menu_cpg_browser.tpl

```diff
@@ -0,0 +1,273 @@
+<div class="col-sm-12">
+  <div class="row">
+    <div id="tabs">
+      <ul class="nav nav-tabs">
+        <li class="statsTab"><a class="statsTabLink" href="/genomic_browser/">CNV</a></li>
```

Comment:
I have just waiting for the profile tab to exist. This will be part of the rebasing.